transformations/transformations.js

```diff
@@ -26,14 +26,25 @@ function formatSeconds(input) {
   return time;
 }
 
-export const transformation = {
+const transformation = {
   hero_id: ({ field, constants }) => `${HOST_URL}${constants.heroes[field] ? constants.heroes[field].img : field}`,
   radiant_win: ({ field, match }) => (isRadiant({ playerSlot: match.player_slot }) === field ? 'W' : 'L'),
   game_mode: ({ field, constants }) => (constants.game_mode[field] ? constants.game_mode[field].name : field),
   start_time: ({ field }) => moment.unix(field).fromNow(),
   duration: ({ field }) => formatSeconds(field),
 };
 
+const transformMatchItem = ({ field, constants }) => {
+  if (field === 0) {
```

Comment:
This is a function we pass into the item_n transformations. So the items that are displayed in match table are item_0, item_1, ... , item_5 (for the 6 slots). That item number is the number in the item table in constants. However, if there is no item, the value is 0. So I am giving the field.display a false value and returning so I don't try to look up that item in the constants and also don't display that item's picture.